frontend/src/locales/zh-CN.json

```diff
@@ -1410,6 +1410,7 @@
       "external-secret-aws": "AWS Secrets Manager",
       "external-secret-gcp": "GCP Secret Manager"
     },
+    "database-region": "数据库区域",
```

Comment:
```suggestion
    "database-region": "数据库地域",
```

Reply:
https://docs.aws.amazon.com/zh_cn/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html

Reply:
as long as we're consistent with their term.